src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -982,7 +982,15 @@ Namespace Microsoft.CodeAnalysis.Semantics
         Private Function CreateBoundBlockOperation(boundBlock As BoundBlock) As IBlockStatement
             Dim statements As Lazy(Of ImmutableArray(Of IOperation)) = New Lazy(Of ImmutableArray(Of IOperation))(
                 Function()
-                    Return boundBlock.Statements.Select(Function(n) Create(n)).Where(Function(s) s.Kind <> OperationKind.None).ToImmutableArray()
+                    Return boundBlock.Statements.Where(
+                            Function(statement)
+                                ' Don't include the method/operator/accessor declaration in the block
+                                Return statement.Syntax.Kind() <> SyntaxKind.OperatorStatement AndAlso
+                                       statement.Syntax.Kind() <> SyntaxKind.FunctionStatement AndAlso
+                                       statement.Syntax.Kind() <> SyntaxKind.GetAccessorStatement AndAlso
+                                       statement.Syntax.Kind() <> SyntaxKind.SetAccessorStatement
```

Comment:
It doesn't, but in removing the NoneOperation filter from C# I removed it from VB as well. I haven't gone through the list of outstanding statement types yet to see if this affects any of others in VB, but I'll try to do so by discussion Thursday.